### PR TITLE
[chore][connector/spanmetrics] Remove albertteoh as codeowner

### PR DESCRIPTION
**Pull request by albertteoh**
**Description:**
Move @albertteoh to emeritus codeowner.

### CODE REVIEW

File: connector/spanmetricsconnector/metadata.yaml [hunk 1]
```diff
@@ -6,7 +6,7 @@ status:
     alpha: [traces_to_metrics]
   distributions: [contrib, sumo, grafana, splunk, liatrio]
   codeowners:
-    active: [albertteoh, portertech]
+    active: [portertech]
```
Comment (crobert-1) [inline connector/spanmetricsconnector/metadata.yaml:9]:
```suggestion
    active: [portertech]
    emeritus: [albertteoh]
```
`make generate` may need to be run after this change as well.

**Review by crobert-1 — APPROVED** [review]
Sorry to hear it, thanks for all of your help here!

**Review by djaglowski — APPROVED** [review]

**Comment by atoulme** [comment]
@albertteoh will you also move to emeritus for https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/spanmetricsprocessor ?

**Comment by albertteoh** [comment]
> will you also move to emeritus for https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/spanmetricsprocessor ?

Yup, sure thing:

https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/30815

**Pull request merged**